docs/sources/examples/mongodb.md

```diff
@@ -133,19 +136,19 @@ as daemon process(es).
 
     # Basic way
     # Usage: docker run --name <name for container> -d <user-name>/<repository>
-    $ docker run --name mongo_instance_001 -d my/repo
+    $ docker run -p 27017:27017 --name mongo_instance_001 -d my/repo
```

Comment:
I'm not sure we should publish to a fixed port on the host here, given that the text above says "we can run one *or more* instances" and the usage explains to get the (random) port number from `docker ps`